Bottom: 8366bf1c1029c0a297a281677772f089895218e9
Top:    1d1da945e49647331f04f82a45cc353e9541a6ae
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-01-11 15:21:46 -0800

Refresh of pre-udp

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 6882528..9b6c667 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -454,6 +454,7 @@ void rs_configure(void)
 
 	if (ucma_init())
 		goto out;
+	ucma_ib_init();
 
 	if ((f = fopen(RS_CONF_DIR "/polling_time", "r"))) {
 		(void) fscanf(f, "%u", &polling_time);
